Ratings_MVP/Ratings_MVPTests/Tests/AssemblyModuleBuilderTests.swift

```diff
@@ -0,0 +1,36 @@
+// AssemblyModuleBuilderTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import Ratings_MVP
+import XCTest
+
+final class AssemblyModuleBuilderTests: XCTestCase {
+    // MARK: - Private Properties
+
+    private var assemblyModuleBuilder: AssemblyBuilderProtocol?
+    var mockMovie = MockMovie()
+    var router = MockMoviesRouter()
+
+    // MARK: - Public Methods
+
+    override func setUpWithError() throws {
+        assemblyModuleBuilder = MockAssemblyModuleBuilder()
+    }
+
+    override func tearDownWithError() throws {
+        assemblyModuleBuilder = nil
+    }
+
+    func testMakeMoviesListScreen() {
+        let movieListScreen = assemblyModuleBuilder?.makeMoviesListScreen(router: router)
+        XCTAssertTrue(movieListScreen is MoviesListViewController)
```

Comment:
```suggestion
        XCTAssertTrue(movieListScreen is MoviesListViewController)
```
а если другой контроллер сюда написать тест сломается? У тебя же даже в шоке этот тип контроллера на выходе как будто